hw/vendor/lowrisc_ibex/ibex_core.core

```diff
@@ -69,9 +69,6 @@ targets:
 
   lint:
     filesets:
-      # Note on Verilator waivers:
-      # You *must* include the waiver file first, otherwise only global waivers
-      # are applied, but not file-specific waivers.
       - tool_verilator ? (files_lint_verilator)
```

Comment:
Please don't change code in the vendor directory, this needs to go through Ibex.

Reply:
Sure, will upstream to Ibex.